sklearn/linear_model/tests/test_ridge.py

```diff
@@ -393,6 +393,40 @@ def _make_sparse_offset_regression(
     return X, y
 
 
+@pytest.mark.parametrize(
+    'solver', ['cholesky', 'sag', 'sparse_cg', 'lsqr', 'saga', 'ridgecv'])
+@pytest.mark.parametrize(
+    'n_samples,dtype,proportion_nonzero',
+    [(20, 'float32', .1), (40, 'float32', 1.), (20, 'float64', .2)])
+@pytest.mark.parametrize('sparse_X', [True, False])
+@pytest.mark.parametrize('seed', np.arange(300))
```

Comment:
300 seeds seems a bit too much.

Reply:
my apologies about that, I had wanted to check with many seeds locally and
forgot to reduce the number of seeds afterwards. I set it to 3 now